cmd/thermal-recorder/throttledRecorder_test.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/TheCacophonyProject/lepton3"
+	"github.com/stretchr/testify/assert"
+)
+
+const THROTTLE_FRAMES int = 27
+const MIN_FRAMES_PER_RECORDING int = 9
+const SPARSE_LENGTH int = 18
+
+var baseRecorder CountWritesRecorder = CountWritesRecorder{}
```

Comment:
The bit after the equals is unnecessary here. This line has exactly the same result:

```go
var baseRecorder CountWritesRecorder
```
In Go, each type has a zero value and each variable of a type is guaranteed to be initialised to the type's zero value.

Reply:
That said, given that it's being used a pointer this might be a more appropriate line:
```go
var baseRecorder = new(CountWritesRecorder)
```
(and then remove the ampersands where it's referred to)